app/browser/reducers/ledgerReducer.js

```diff
@@ -503,6 +503,15 @@ const ledgerReducer = (state, action, immutableAction) => {
         state = ledgerApi.onFetchReferralHeaders(state, action.get('error'), action.get('response'), action.get('body'))
         break
       }
+    case appConstants.APP_ON_LEDGER_FUZZING:
+      {
+        state = ledgerState.setAboutProp(state, 'status', ledgerStatuses.FUZZING)
```

Comment:
Should we set this prop after `newStamp` has been validated on line `510`?

Reply:
I think that as soon as fuzzing is triggered we should display notification, even if something was not send over correctly. wdyt?

Reply:
That sounds fine yeah, I just assumed that if the check failed the process to that point would be so quick it may not be worth it.